### PR TITLE
Automation for creating a sqlite package for redistributing sqlite3.dll

### PR DESCRIPTION
**Pull request by natemcmaster**
Here is a KoreBuild friendly setup that builds a sqlite package with native windows components only.

cc @divega @moozzyk @eilon @davidfowl @bricelam

### CODE REVIEW

**Comment by natemcmaster** [comment]
cc @mistachkin

File: tools/PackageBuilder/Program.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                {
+                    Directory.SetCurrentDirectory(slnRoot);
+                }
+                
+                try 
+                {
+                    Directory.SetCurrentDirectory("src/libsqlite3.native");
+                    NativePackage.Build(SqliteVersion);
+                }
+                finally
```
Comment (moozzyk) [inline tools/PackageBuilder/Program.cs:23]: nit: could you avoid all nested try/catch and have one finally after the exception handler below?

**Comment by moozzyk** [comment]
I think you might be missing some legal stuff (e.g. we have [this](https://github.com/aspnet/libuv-package/blob/dev/content/thirdpartynotices.txt)) in libuv. @Eilon will probably know what's needed.

Other than that LGTM

**Comment by natemcmaster** [comment]
Updated. I added the "public domain" to the copyright. Also, it looks like we won't be running this on our own build server. I've stripped out the korebuild part and updated to the package names.

**Comment by bricelam** [comment]
LGTM

**Pull request merged**